hack/test/e2e-aws.sh

```diff
@@ -78,8 +78,7 @@ function setup {
   ## TODO: update to talos-systems once merged
   ${CLUSTERCTL} config cluster ${NAME_PREFIX} \
     --kubeconfig /tmp/e2e/docker/kubeconfig \
-    --from https://github.com/talos-systems/cluster-api-templates/blob/main/aws/standard/standard.yaml > ${TMP}/cluster.yaml
```

Comment:
can we revert this change, merge this PR

then refactor templates to use versions via folders, and flip the template here to 0.2 version with cloud-provider external from the config as a separate PR?

Reply:
Done